internal/pkg/develop/plugin/template/delete.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
 )
 
-func Delete(options map[string]interface{}) (bool, error) {
+func Delete(options configmanager.RawOption) (bool, error) {
```

Comment:
typo
configmanager.RawOption -> configmanager.RawOptions

I suggest you to use `cmd+shift+f` to search `configmanager.RawOption)` to find all typos.

<img width="712" alt="image" src="https://user-images.githubusercontent.com/36830265/196352577-8e5150c9-fc56-4391-9a2e-6f7e182f9609.png">